src/swap/components/SwapAmountInput.test.tsx

```diff
@@ -182,10 +197,49 @@ describe('SwapAmountInput', () => {
         swappableTokens={mockSwappableTokens}
       />,
     );
-    const dropdown = screen.getByText('TokenSelectDropdown');
+    const dropdown = screen.getByText(/TokenSelectDropdown/i);
     expect(dropdown).toBeInTheDocument();
   });
 
+  it('should correctly select a token from the dropdown using mouse and keyboard', () => {
+    const useSwapContextMock = useSwapContext as vi.MockedFunction<
+      typeof useSwapContext
+    >;
+    useSwapContextMock.mockReturnValue(mockContextValue);
+
```

Comment:
remove spaces, as they make hard to read dozends of tests at the same time.

Reply:
done